File: ios/SimpleStrongBiometrics.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(SimpleStrongBiometrics, NSObject)

RCT_EXTERN_METHOD(getItem: (NSString *)key options:(NSDictionary *)options resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)

RCT_EXTERN_METHOD(setItem: (NSString*)key value:(NSString*)value options:(NSDictionary *)options resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)

RCT_EXTERN_METHOD(deleteItem: (NSString *)key options:(NSDictionary *)options resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)

RCT_EXTERN_METHOD(hasStrongBiometricEnabled: (RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)

@end

File: ios/SimpleStrongBiometrics.swift
@objc(SimpleStrongBiometrics)
class SimpleStrongBiometrics: NSObject {

    
    @objc func getItem(_ key: String, options: NSDictionary, resolver resolve: RCTPromiseResolveBlock, rejecter reject: RCTPromiseRejectBlock) -> Void {
        resolve("No implemented yet")
    }
    
    
    @objc func setItem(_ key: String, value: String, options: NSDictionary, resolver resolve: RCTPromiseResolveBlock, rejecter reject: RCTPromiseRejectBlock) -> Void {
        resolve(nil)
    }
    
    @objc func deleteItem(_ key: String, options: NSDictionary, resolver resolve: RCTPromiseResolveBlock, rejecter reject: RCTPromiseRejectBlock) -> Void {
        resolve(nil)
    }
    
    @objc func hasStrongBiometricEnabled(_ resolve: RCTPromiseResolveBlock, rejecter reject: RCTPromiseRejectBlock) -> Void {
        resolve(false)
    }
}
